src/scss/components/checkout/widget/_widget.scss

```diff
@@ -72,15 +72,21 @@
     margin-top: remCalc(20px);
     padding: 1rem;
 
+    @media screen and (max-width: 414px) {
```

Comment:
This seems like a magic number. Should be looking at using a more standard screen/viewport size? 

Thoughts?

Reply:
This size was taken from iphone 6/7/8 plus. It was added because between  375px and 414px, expiration date and card fields were still very close and overlapping.

Reply:
is there any possibility of creating a variable for this magic number? so this way it's self documented

Reply:
Yes. It is already added. Thanks a lot.